src/Methods.cs

```diff
@@ -617,6 +637,41 @@ private IEnumerable<string> GetProjectReferencesFromPackagesConfig(string projec
             return Array.Empty<string>();
         }
 
+        /// <summary>
+        /// Gets project references from a project.assets.json file.
+        /// </summary>
+        /// <param name="assetsPath">The assets file</param>
+        /// <returns></returns>
+        private IEnumerable<string> GetProjectReferencesFromAssetsFile(string assetsPath)
```

Comment:
Can you please add a unit test for this method?

Reply:
Added `GetProjectReferencesFromAssetsFile_Should_Resolve_Transitive_Assets`